lib/force.go

```diff
@@ -342,6 +342,10 @@ func (f *Force) GetCodeCoverage(classId string, className string) (err error) {
 	var result ForceQueryResult
 	json.Unmarshal(body, &result)
 
+	if len(result.Records) == 0 {
+		ErrorAndExit("class not found")
+	}
+
```

Comment:
Functions in lib should return an error rather than exit so calling applications can handle the error.

Reply:
Fixed. Should we do something about the other methods that call `ErrorAndExit` in lib/force.go then?